pkg/datastore/applicationstore.go

```diff
@@ -35,6 +36,30 @@ func (a *applicationCollection) Factory() Factory {
 	}
 }
 
+func (a *applicationCollection) GetStoredFileNames(id string) []string {
+	return []string{
+		fmt.Sprintf("%s_api", id),
```

Comment:
Should object files have an explicit extension such as `.json`?
Which format the data will be stored in?

Reply:
I think the file extension should be defined and controlled in filedb package, not in the datastore layer, wdyt

Reply:
I changed my mind, should make the extension here to reduce the complexity 👍

Reply:
Updated by https://github.com/pipe-cd/pipecd/pull/3272/commits/399adb1bd1aefc0f4fa1c110c4ead941cda0e190 🙏

Reply:
Okay. 👍